frontend/src/metabase/admin/databases/containers/DatabaseEditApp.unit.spec.js

```diff
@@ -53,9 +59,43 @@ async function setup({ cachingEnabled = false } = {}) {
   });
 
   await waitForElementToBeRemoved(() => screen.queryByText("Loading..."));
+
+  return { mockEventListener };
 }
 
 describe("DatabaseEditApp", () => {
+  describe("Database connections", () => {
+    afterEach(() => {
+      jest.restoreAllMocks();
+    });
+
+    it("should trigger beforeunload event when database connection is edited", async () => {
+      const { mockEventListener } = await setup();
+
+      const databaseForm = await screen.findByTestId("database-name-field");
```

Comment:
Can we do `screen.findByLabelText("Display name")` instead?

Reply:
and remove than data-testid

Reply:
Yep, sounds good!